zuul-ci/run-tests.yaml

```diff
@@ -0,0 +1,9 @@
+- hosts: all
+  roles:
+    - repo-setup
```

Comment:
This repo-setup role creates the zuul-built.repo file: https://pagure.io/zuul-distro-jobs/blob/master/f/roles/repo-setup/tasks/main.yaml